products/compute/api.yaml

```diff
@@ -1945,13 +1945,23 @@ objects:
 %>
 <%= indent(compile('products/compute/instance_scheduling.yaml'), 6) %>
 <%= indent(compile('products/compute/instance_serviceaccounts.yaml'), 6) %>
-      - !ruby/object:Api::Type::String
+      - !ruby/object:Api::Type::Enum
         name: 'status'
         description: |
           The status of the instance. One of the following values:
           PROVISIONING, STAGING, RUNNING, STOPPING, SUSPENDING, SUSPENDED,
           and TERMINATED.
-        output: true
```

Comment:
Drive-by: `api.yaml` should reflect the API, and this should be done as an override.

Reply:
Is this not what the API reflects? The status can be changed through APIs

This seems like one of the weird places where the API isn't descriptive enough. The status reflects an action (start, stop) that you've taken on an instance. In this case, we're using status as both an end-state and a trigger for actions.

Reply:
Hmm oh no. Makes sense. Do you mind adding a comment describing why this shows `[Output only]` in the docs but that isn't true?

Reply:
Done.